### DISCUSSION TITLE
Chrome confirmation messages bugging i3

### DISCUSSION THREAD

**Discussion by vbartle** [i3/i3#5961] [Q&A]
I can't really take a screenshot of this, but to replicate: (1) open a tab I know always asks for confirmation before being closed, e.g. a jupyter notebook with an unsaved change (2) ctrl+w close, confirmation message doesn't show, (3) if I hit enter the tab closes and if I hit escape the tab unfreezes. 

This is partly a chrome problem, but I imagine there's a config or daemon I'm supposed to setup to control this? Any help is appreciated.

**Comment by vbartle** [comment]
? Is there something I can provide to make it easier to get help with this?

**Comment by KebabLord** [comment]
do you live the same problems when you run these commands on dev console?:
```js
alert('This is an alert.');
prompt('This is a prompt.');
confirm('This is a confirmation.');
```
Do you live the same problem when you disable "Use graphics acceleration when available" setting?

> but I imagine there's a config or daemon I'm supposed to setup to control this?

Using a compositor such as "compton" or "picom" fixes many problems such as transparency, floating etc on tilling wms. Can you try if that'll work?

**Reply by vbartle** [reply]
Thank you for your response.

(a) alert/prompt/confirm does not have a problem
(b) turning off graphics accelerations does not resolve problem
(c) picom seems to have mildly changed the problem, but the glitching persists. This time I was able to take a screenshot, since the glitching is now viewable. 
<img width="390" alt="2024-03-31_19-00" src="https://github.com/i3/i3/assets/14290184/a02f37cb-70dd-49fc-a3fd-e84e0993d9f2">


I think you're right that there's some sort of graphics card problem here -- though I don't experience other gpu-type problems.

Seems this problem doesn't happen on other browsers though, so I believe this is chrome-specific, although again I don't get this outside i3. Closing anyways.

**Reply by KebabLord** [reply]
i guess this is a general i3 bug, as stated here #3537 Sadly it's closed without a proper solution.
People say it has something to do with GPU, but some people gets it fixed on other WMs, some live the same problem even after switching.

The additional workarounds you can try are:
- Instead of picom, try compton to see if that works
- Start the i3 from tty by running startx, by not using any display manager
- Maybe you have an existing DM package which creates conflict, try removing KDE, Plasma packages if they exist 

Also i wonder if people living this problem has a common denominator, can you share the information about the following:
- What is the i3 version, GPU and Distro you have?
- Did you make a clean install + i3, or switched from another WM/DM, if yes from which one?
- What login manager you are using?
- are you living this problem with older chrome versions too?
